libkpod/stats.go

```diff
@@ -0,0 +1,161 @@
+package libkpod
+
+import (
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"strings"
+
+	"github.com/opencontainers/runc/libcontainer"
+)
+
+// ContainerStats contains the statistics information for a running container
+type ContainerStats struct {
+	Container   string
+	CPU         float64
+	cpuNano     uint64
+	systemNano  uint64
+	MemUsage    uint64
+	MemLimit    uint64
+	MemPerc     float64
+	NetInput    uint64
+	NetOutput   uint64
+	BlockInput  uint64
+	BlockOutput uint64
+	PIDs        uint64
+}
+
+// GetContainerStatsFollow gets the running stats for a running container continually
+func (c *ContainerServer) GetContainerStatsFollow(ids []string, all bool, statsChan chan []*ContainerStats, times int) error {
```

Comment:
Actually, doing this will require quite a bit of refactoring.  the slice of ids is a user-supplied list used in the filters.  If I move all of that outside of the code, I'll have to have a loop in cmd/kpod that
a) gets a list of images matching the filter (in case it's changed since last time
b) calls GetContainerStats(ctr *oci.Container) to get the stats for a container
c) prints the container stats
d) repeats a) and b) for each container in the list
I would also have to do a-d outside of the loop the first time because some of the stats are based on a change from the previous value

So I could do it, but it would remove all concurrency from the command and move a lot of logic into cmd/kpod.  Do you still want me to make the method accept a single container?